appshell/mac/brackets.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+TARGET_FILE=$0
+BRACKETS_APP_NAME="Brackets.app"
+
+TARGET_PATH="$(readlink "$TARGET_FILE")"
```

Comment:
So if I'm reading this correctly, `$0` will be the symlink (`/usr/local/bin/brackets`), and `TARGET_PATH` will be the actual location of this .sh file, inside the Brackets.app bundle?  Could we add a comment here explaining that?